content/index.md

```diff
@@ -31,6 +31,35 @@ the assets on Aurora are the same as Ethereum. For example, account balances
 in Aurora's EVM are denominated in the same Ether (ETH) as on Ethereum
 itself.
 
+# Aurora Developer cheatsheet
+
+### Gas fees and API Keys
+At the moment we cover all the gas costs on the mainnet. This makes Aurora a good target for various DDoS and Spam attacks.
+To fight those, we have API keys. There's a public UI in development for developers to be able to get those, but for the time being those have to be issued manually. To request an API key, message Anton ([telegram](https://t.me/antonpaisov), [email](mailto:anton@aurora.dev)) or Ahmed ([telegram](https://t.me/theZeroX)).
+To generate an API key we will need an email address to associate with it, so please include it in the message.
+
+To use the key simply append it to the network host address. So `https://mainnet.aurora.dev` becomes `https://mainnet.aurora.dev/YOUR_API_KEY`
+
+### Maximum gas per transaction (a monthly updated section)
+The main issue we have on Aurora and we're constantly working on improving is the maximum gas per transaction limit.
+The nature of the issue comes from the fact that the block time on NEAR is 1 second (for comparison it's 14 seconds on Ethereum).
+Naturally the transaction must fit in a block.
```

Comment:
I don't get the purpose of this message, but if we are talking about most cases, let's use the following phrasing:

```suggestion
Usually, the transaction must fit in a block.
```